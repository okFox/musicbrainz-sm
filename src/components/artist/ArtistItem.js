import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';
import styles from './ArtistItem.css';

const ArtistItem = ({ artistName, artistId }) => (

  <Link to={`/albums/${artistId}/${artistName}`}>
    <li className={styles.artistItem}>
      <h1>{artistName.replace('-', '')}</h1>
    </li>
  </Link>
);

ArtistItem.propTypes = {
  artistName: PropTypes.string,
  artistId: PropTypes.string
};

export default ArtistItem;
