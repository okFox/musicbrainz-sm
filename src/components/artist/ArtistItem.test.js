import React from 'react';
import { shallow } from 'enzyme';
import ArtistItem from './ArtistItem';

describe('ArtistItem component', () => {
  it('renders ArtistItem', () => {
    const wrapper = shallow(<ArtistItem artistId="1"  artistName="name" />);
    expect(wrapper).toMatchSnapshot();
  });
});
