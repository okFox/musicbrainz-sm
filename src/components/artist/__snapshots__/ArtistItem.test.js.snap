// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistItem component renders ArtistItem 1`] = `
<Link
  to="/albums/1/name"
>
  <li
    className="artistItem"
  >
    <h1>
      name
    </h1>
  </li>
</Link>
`;
